### PR TITLE
Moved property setting to a separate function

### PR DESCRIPTION
**Pull request by zefciu**
We want to set device properties from other places than just save() of
the device form. Thus we move the logic to a separate function.

### CODE REVIEW

**Comment by andrzej-jankowski** [comment]
:ok:

File: src/ralph/discovery/models_device.py [hunk 1]
```diff
@@ -925,6 +925,30 @@ def save(self, sync_fields=True, *args, **kwargs):
         # the reason (for more, see the source code of SavePrioritized).
         return super(Device, self).save(*args, **kwargs)
 
+    def set_property(self, symbol, value, user):
+        from ralph.business.models import RoleProperty, RolePropertyValue
+        try:
+            p = self.venture_role.roleproperty_set.get(symbol=symbol)
+        except RoleProperty.DoesNotExist:
+            p = self.venture.roleproperty_set.get(symbol=symbol)
```
Comment (ar4s) [inline src/ralph/discovery/models_device.py]: why not ``p, _ = self.venture.roleproperty_set.get_or_create(symbol=symbol)``?

Comment (andrzej-jankowski) [inline src/ralph/discovery/models_device.py]: self.venture_role vs self.venture

Comment (andrzej-jankowski) [inline src/ralph/discovery/models_device.py]: It's ok here.

Comment (zefciu) [inline src/ralph/discovery/models_device.py]: I don't know why. That's not my code :D

**Comment by ar4s** [comment]
:ok:

**Pull request merged**